src/main/scala/org/globalnames/formatters/Details.scala

```diff
@@ -12,8 +12,7 @@ object Details {
     case ScientificName(_, Some(ng), _) => format(ng)
   }
 
-  def format(namesGroup: NamesGroup): JValue =
-    JArray(namesGroup.name.map(format).toList)
+  def format(namesGroup: NamesGroup): JValue = namesGroup.name.map(format).toList
```

Comment:
What do you think about keeping lilnes at 80 columns? http://docs.scala-lang.org/style/indentation.html#line_wrapping